alfresco-transform-tika/alfresco-transform-tika-boot/pom.xml

```diff
@@ -97,7 +97,7 @@
         <dependency>
             <groupId>org.bouncycastle</groupId>
             <artifactId>bcmail-jdk15on</artifactId>
-            <version>1.68</version>
+            <version>1.64</version>
```

Comment:
Downgrade ?

Reply:
Hmm, not sure how that entered... I'll fix now